### PR TITLE
Cloud 0.19 release notes

### PR DESCRIPTION
**Pull request by paolaperaza**
Starting a draft here for 0.19 Cloud release notes

### CODE REVIEW

**Comment by paolaperaza** [comment]
@vishwas-astro Wanna take a quick look at this? Just pushed up some changes to reflect latest fixes included in 0.19. For reference, I'm looking at these issues:h ttps://github.com/astronomer/issues/issues?q=is%3Aissue+milestone%3A0.19+is%3Aclosed

I did NOT include GH issues that were a part of the 0.19 milestone but not "Closed," and I also didn't include NATS work. Is any of that worth mentioning? Let me know what you think.

File: cloud/stable/06_resources/01_release-notes.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ navTitle: "Release Notes"
...
+## Astronomer v0.19 Release Notes
+
+Release Date: August 27, 2020
+
+### v0.19.0
+
+#### Bug Fixes and Improvements
+
+- Improvement: Display toast message in Astronomer UI following Deployment deletion
+- Improvement: Replace `workspaceUuid` with `deploymentUuid` in arg for `deploymentVariables` Houston API mutation
```
Comment (vishwas-astro) [inline cloud/stable/06_resources/01_release-notes.md:16]: I would include another improvement here saying that our create, update and delete deployment operations are now implemented in an event driven way and will help towards further reliable deployments or some such language. 

wdyt @schnie?

Comment (paolaperaza) [reply cloud/stable/06_resources/01_release-notes.md:16]: @vishwas-astro Good call. I added: 

> Improvement: The 'Create', 'Update' and 'Delete' Deployment operations are now implemented in an event-driven framework optimized for reliability

**Comment by paolaperaza** [comment]
@vishwas-astro Alright, merging this so I can send the email but we can always iterate. If there's anything else you'd suggest changing - lemme know and I'll create another PR. Thanks for reviewing!

**Pull request merged**